lib/transmission-rss/feed.rb

```diff
@@ -12,6 +12,8 @@ def initialize(config = {})
         @url = URI.encode(config['url'] || config.keys.first)
 
         @download_path = config['download_path']
+		
+        @validate_cert = config['validate_cert']
```

Comment:
Isn't it neccessary to set it to `true` if it is not set in the config? Seems like currently no cert is validated. Could you check on this again, please?

The condition could look like this, probably:

    @validate_cert = config['validate_cert'].nil? || config['validate_cert']